sass/mixins/btn.scss

```diff
@@ -0,0 +1,81 @@
+@import "palette.scss";
+@import "dimensions.scss";
+@import "typography.scss";
+
+//dimensions
+$button-padding-vert: $base-spacing-unit + 1px;
+$button-padding-horiz: $large-spacing-unit;
+
+$button-color: white;
+
+@mixin btn($button-bg: $color-500-grey) {
+  //variables
+  $button-hover-bg: darken($button-bg, 8%);
+  $button-active-bg: darken($button-bg, 16%);
+  //TODO (jp-18-3-15) need to check this is the right transform
+  $button-disabled-bg: lighten($button-bg, 16%);
+  $button-disabled-color: lighten($button-bg, 32%);
+
+
+  //resets (for <button></button> elements
```

Comment:
Im going to leave it here because if `$button-color` becomes configurable we are safe.